sdk/confidentialledger/azure-confidentialledger/sdk_packaging.toml

```diff
@@ -0,0 +1,2 @@
+[packaging]
+auto_update = false
```

Comment:
can you make your sdk packaging toml file like [this](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/sdk_packaging.toml)? i think that will get rid of some of the errors you're seeing in ci